raiden/tests/conftest.py

```diff
@@ -59,11 +59,11 @@ def pytest_addoption(parser):
     )
 
     parser.addoption(
-        '--synapse-base-port',
```

Comment:
This probably needs improvement, because a value `None` should allowed for mac users.

Reply:
I'd say don't worry about it. This should only really be necessary to use when running tests in parallel.